wordpress/wp-content/plugins/memberful-wp/src/contrib/wp-recipe-maker.php

```
@@ -0,0 +1,827 @@
+    $adding_preview = false;
+
+    if ( '' === trim( $preview ) ) {
+      return $content;
+    }
+
+    if ( preg_match( '/<div class=["\']memberful-global-marketing-content["\']>/', $content ) ) {
+      return preg_replace( '/<div class=["\']memberful-global-marketing-content["\']>/', $preview . '$0', $content, 1 );
+    }
+
+    return $preview . $content;
+  }
```

PHPMD (2.15.0)

[error] The method add_locked_recipe_preview_to_protected_content is not named in camelCase. (undefined)

(CamelCaseMethodName)